UnityProject/Assets/Scripts/HealthV2/Living/CirculatorySystem/BodyPartBlood.cs

```diff
@@ -123,11 +123,10 @@ public partial class BodyPart
 		/// </summary>
 		public void BloodInitialise()
 		{
-			if (bloodType == null)
+			if (bloodType == null) //TODO
```

Comment:
what is left to do?

Reply:
needs to be moved over to the  Reagent circulated component when Turning on new system

Reply:
add that to the todo comment. Even if it is a note for yourself, if it enters the codebase it needs to be understood by everyone

Reply:
ok